src/fragments/lib-v1/storage/js/upload.mdx

```diff
@@ -163,4 +163,28 @@ Uploads that were initiated over one hour ago will be cancelled automatically. T
 
 </Callout>
 
+## Browser uploads
+
+Upload a file in the browser:
+
+```javascript
+import { uploadData } from 'aws-amplify/storage'; 
+
+const uploadDataInBrowser = async () => {
+		if (event?.target?.files) {
+			const file = event.target.files[0];
+			uploadData({
+				key: filename,
+				data: file,
+			});
+		}
+	};
+
+<h2>Upload File</h2>
+<input
+	type="file"
+	onChange={() => uploadDataInBrowser(event)}
+/>
```

Comment:
Should this markup be in a different code snippet since it's not valid Javascript?

Reply:
Yeah that makes sense.